sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_export/_base.py

```diff
@@ -62,22 +66,35 @@ class ExportResult(Enum):
 class BaseExporter:
     """Azure Monitor base exporter for OpenTelemetry."""
 
-    def __init__(self, **kwargs: Any) -> None:
+    def __init__(
+        self,
+        *,
```

Comment:
If the args are never used, do we really want to put *, here?

Reply:
This behaviour is identical to the existing `**kwargs` syntax, just declares the contents of kwargs more specifically to support type hinting and a better IDE/intellisense experience.

Reply:
Got it.